src/sourmash/sbt.py

```diff
@@ -286,6 +287,8 @@ def add_node(self, node):
             c1 = self.children(p.pos)[0]
             self._leaves[c1.pos] = node 
             node.update(n)
+        else:
+            assert 0
```

Comment:
maybe be more explicit why this assertion is here, it will be hard to figure out in the future =]